### PR TITLE
MF-997 - Add twins service swagger file

### LINKED ISSUE #997: Write swagger file for Twins service
Write a swagger specification that describes the HTTP CRUD for the Twins service.

### PR DESCRIPTION
**Pull request by darkodraskovic**
Signed-off-by: Darko Draskovic <darko.draskovic@gmail.com>

Resolves #997

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

**Review by manuio — DISMISSED** [review]
LGTM

**Review by blokovi — DISMISSED** [review]

File: twins/swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+      description: |
+        Retrieves a list of managed twins. Due to performance concerns, data
+        is retrieved in subsets.
+      tags:
+        - twins
+      parameters:
+        - $ref: '#/parameters/Authorization'
+        - $ref: '#/parameters/Limit'
+        - $ref: '#/parameters/Offset'
+        - $ref: '#/parameters/Metadata'
```
Comment (manuio) [inline twins/swagger.yaml]: looks like `name` is missing here

**Review by manuio — CHANGES_REQUESTED** [review]

File: twins/swagger.yaml (hunk 1 above)
Comment (darkodraskovic) [reply twins/swagger.yaml]: No, everything is ok. You can check it by pasting it here https://editor.swagger.io/

Names are defined in `parameters` section.

Comment (manuio) [reply twins/swagger.yaml]: you can search by name: https://github.com/mainflux/mainflux/blob/master/twins/api/http/transport.go#L167

Just add - $ref: '#/parameters/Name'

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by drasko — DISMISSED** [review]
LGTM

**Review by manuio — DISMISSED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Review by manuio — APPROVED** [review]
LGTM

**Pull request merged**